tools/msvs/find_python.cmd

```diff
@@ -57,7 +57,7 @@ set "need_path_ext="
 exit /b 0
 
 :check-python
-%~1 -V
+%* -V
```

Comment:
@lux01 can you comment on why this change is needed?

Reply:
This is needed when the path to `python.exe` has a space in it. Python 3.10 when installed for all users defaults to `C:\Program Files\Python310\python.exe` which fails without this change:

```
PS C:\source\node> .\vcbuild.bat debug vs2019 x64 dll
Looking for Python
Python found in C:\Program Files\Python310\\python.exe
'C:\Program' is not recognized as an internal or external command,
operable program or batch file.
Not an executable Python program
Could not find Python.
```